pkg/web/templates/components/transaction_rejection.html

```diff
@@ -14,6 +14,7 @@ <h3 class="font-bold text-xl">Reject Transaction</h3>
             <select id="transaction-rejection" name="code"
             class="!bg-gray-50 border !border-gray-300 !text-gray-900 rounded-lg focus:!ring-blue-500 focus:!border-blue-500 block w-full p-2.5">
             <option value="REJECTED">Rejected</option>
+            <option value="BVRC999">Request could not be processed by recipient (Alias: Sygna BVRC Rejected Code)</option>
```

Comment:
```suggestion
```

I'm not sure Sygna is still around -- but Envoy customers probably won't be using Sygna so I don't think we have to expose the Sygna codes to them.